### PR TITLE
Separate charts [BREAKING CHANGE TO HELM CHARTS]

### LINKED ISSUE #213: Helm upgrade does not upgrade CRDs
Current method of specifying CRDs is ignored when upgrading helm release.

See: https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Possible solution:

* Move CRDs to `./charts/kubeai/templates`
* Move model up a level to be its own chart

Bonus:

* Template a catalog of individual `<model-name>.yaml` files that could be installed with `kubectl apply -f https://...` directly. Probably makes sense to do this with the models chart.

### PR DESCRIPTION
**Pull request by nstogner**
Split the model catalog in `charts/kubeai/` into its own chart (`charts/models/`). Move CRDs to `templates/` to allow users to `helm upgrade` CRDs.

Fixes #213

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -58,23 +58,28 @@ helm repo update
...
-EOF
-
 helm upgrade --install kubeai kubeai/kubeai \
-    -f ./helm-values.yaml \
     --wait --timeout 10m
 ```
 
+Install some predefined models.
+
+```bash
```
Comment (samos123) [inline docs/README.md:67]: another alternative could be to use `kubectl apply -f http://`

I kind of like the idea that you always see the full model CRD that is getting applied.

Comment (nstogner) [reply docs/README.md:67]: I like that idea too. Any customizations would likely need to be done with a following kubectl command or a multi-step download file, edit, apply... Thinking of setting `minReplicas: 1`

Comment (nstogner) [reply docs/README.md:66]: Lets punt on this for now.

File: charts/kubeai/Chart.yaml [hunk 2]
```diff
@@ -41,7 +29,7 @@ home: https://www.substratus.ai
 
 maintainers:
 - name: nstogner
-  email: nicholas.stogner@gmail.com
+  url: https://www.linkedin.com/in/nstogner/
```
Comment (samos123) [inline charts/kubeai/Chart.yaml:32]: I think we need an email later if we upload to artifact hub, but fine with removing for now

File: docs/installation/gke.md [hunk 3]
```diff
@@ -39,33 +39,44 @@ TODO: Reference gcloud commands for creating a GKE standard cluster.
...
+## 3. Optionally configure models
 
-Install KubeAI with [Helm](https://helm.sh/docs/intro/install/).
+Optionally install preconfigured models.
 
 ```bash
-helm repo add kubeai https://substratusai.github.io/kubeai/
-helm repo update
+cat <<EOF > kubeai-models.yaml
+models:
```
Comment (samos123) [inline docs/installation/gke.md:74]: should this now be:
```
catalog:
  llama-..:
```

File: docs/installation/gke.md [hunk 4]
```diff
@@ -39,33 +39,44 @@ TODO: Reference gcloud commands for creating a GKE standard cluster.
...
-Define the installation values for GKE.
+Add KubeAI Helm repository.
 
 ```bash
-cat <<EOF > helm-values.yaml
-models:
-  catalog:
-    llama-3.1-8b-instruct-fp8-l4:
-      enabled: true
+helm repo add kubeai https://substratusai.github.io/kubeai/
```
[the remark this answers is not in the document]
Comment (samos123) [reply docs/installation/gke.md:45]: I am fixing this in a follow up commit

File: docs/installation/gke.md (hunk 3 above)
Comment (samos123) [reply docs/installation/gke.md:74]: I am fixing this in a follow up commit

**Review by samos123 — APPROVED** [review]

**Pull request merged**